lms/envs/common.py

```diff
@@ -1248,12 +1248,34 @@
 base_application_js = [
     'js/src/utility.js',
     'js/src/logger.js',
+    'js/my_courses_dropdown.js',
```

Comment:
Note for reviewers: my_courses_dropdown is necessary for the dropdown on the RHS (dashboard, profile, account settings, log out).

string_utils is used for an interpolate helper method.

from.ext is needed for AJAX callbacks.

ie_shim and accessibility_tools just seem like generally good things to have available.